src/BloomExe/ImageProcessing/ImageServer.cs

```diff
@@ -75,20 +77,33 @@ protected override bool ProcessRequest(IRequestInfo info)
 				return false;
 
 			var r = GetLocalPathWithoutQuery(info);
-			if (r.EndsWith(".png") || r.EndsWith(".jpg"))
+
+			// only process these image types
+			if (!r.EndsWith(".png") && !r.EndsWith(".jpg") && !r.EndsWith(".svg")) return false;
+
+			r = r.Replace("thumbnail", "");
+
+			// This happens with the new way we are serving css files
+			if (!File.Exists(r))
+			{
+				var fileName = r;
+				var pos = fileName.LastIndexOfAny(new[] { '\\', '/' });
+				if (pos > -1) fileName = fileName.Substring(pos + 1);
```

Comment:
The doc seems to indicate that it just strips off everything before the last of the two directory separators it recognizes, like your code. Should be tested, of course. It just feels like an unnecessary bit of technical debt that we should be using a private way of extracting the filename from a path (in addition to all the places we use GetFileName().